Dockerfile

```diff
@@ -24,6 +24,7 @@ RUN apk add --no-cache --virtual .build-deps \
 RUN apk add --no-cache \
   jpeg-dev \
   libc-dev \
+  libffi-dev \
```

Comment:
@limed This addition to the Dockerfile is needed to allow `mozilla-django-oidc` to be installed. OK with you from an Ops point of view?

Reply:
This sounds good from my point of view :+1: